frequency : 10

goal_topic          : testGoal
costmap_topic       : /costmap_generator/local_costmap
path_topic          : local_path

robot_frame_id      : base_link